tools/osbuilder/image-builder/image_builder.sh

```diff
@@ -516,14 +565,21 @@ main() {
 		die "Invalid rootfs"
 	fi
 
-	img_size=$(calculate_img_size "${rootfs}" "${root_free_space}" "${fs_type}" "${block_size}")
-
-	# the first 2M are for the first MBR + NVDIMM metadata and were already
-	# consider in calculate_img_size
-	rootfs_img_size=$((img_size - dax_header_sz))
-	create_rootfs_image "${rootfs}" "${image}" "${rootfs_img_size}" \
+	if [ "${fs_type}" == 'erofs' ]; then
```

Comment:
I think that `calculate_img_size` is still needed :thinking:

Reply:
Hmmm.. not quite sure why it's needed, it's useful for ext4/xfs since their mkfs can only generate an empty image.
but mkfs.erofs can generate an image directly, so I think no need to guess the target size first.

Reply:
Hmmm.. not quite sure why it's needed, it's useful for ext4/xfs since their mkfs can only generate an empty image.
but mkfs.erofs can generate an image directly, so I think no need to guess the target size first.